RunningOrder/Story/Models/Configuration.swift

```diff
@@ -14,6 +14,7 @@ struct Configuration {
     var features: [String] = []
     var indicators: [String] = []
     var identifiers: [String] = []
+    var links: [LinkEntity] = []
```

Comment:
Why not just Link ? Instead of LinkEntity?

Reply:
Cause there was a conflict with Swift UI class "Link" but I fixed it (changes in the upcoming commit)

Reply:
aaaah